Upgrade-Guide/op_version.md

```diff
@@ -0,0 +1,29 @@
+
+###op-version
+op-version is the operating version of the Gluster which is running.
```

Comment:
We need to detail out a bit more about what's the need of having op-version. op-version was introduced to ensure gluster bits running with different version of bits do not end up in a problem and backward compatibility issues can be tackled.